agent/tracestate.md

```diff
@@ -0,0 +1,41 @@
+# Trace state
+
+The Hypertrace agents encode vendor specific information in [tracestate header](https://www.w3.org/TR/trace-context/#tracestate-header).
+
+## Format
+
+The tracestate format is `vendorname1=opaqueValue1,vendorname2=opaqueValue2`. The value
+is up to 256 characters long and contains ASCII from `0x20` to `0x7E` excluding `,` and `=`. For more
+detailed information see the [specification](https://www.w3.org/TR/trace-context/#value).
+
+The Hypertrace vendor name is `hypertrace`.
+
+### Value
+
+The Hypertrace tracestate vendor value can encode multiple values. The values are separated by `|`. 
+
+An example:
+* `tracestate: somevendor=opaqueValue1,hypertrace=key1:val1|key2:val2`
+
+#### Capture
+
+The capture feature allows coherently capturing data within a trace. For instance a sampler
+can make use of this capability to capture only some percentage of request and response payloads.
```

Comment:
I see you added this example but how exactly can we achieve this with `cap`?

Reply:
1. sampler makes the decision and sets the cap value
2. instrumentation uses tracestate to capture values OR the data is dropped before reporting  OR there is a filter for adding a data to span

2. depends on the instrumentation API. In OTEL it seems that the instrumentation points will have to look at the tracestate to determine whether data should be captured or not.